boost_libraries.bzl

```diff
@@ -97,7 +97,12 @@ BOOST_LIBRARIES = [
 
     boost_library(
         name = "chrono",
-        deps = ["ratio"],
+        defines = ["BOOST_CHRONO_HEADER_ONLY"],
```

Comment:
If there is no strong reason, can we add the source files instead of using header-only?